src/api/users.iced

```diff
@@ -70,6 +72,29 @@ users.handle_add_data = (req, resp) ->
   }
   req.couch.use('_users').atomic('base', 'do_action', user, action).pipe(resp)
 
+users.reactivate_user = (client, user, callback) ->
+  action = {
+    action: 'u+'
```

Comment:
it should be the same functionality. whether the user already existed and was de-activated, or is a brand new user, the u+ should do the same thing in order to be idempotent.